### PR TITLE
Is/read me update/best practices

### PR DESCRIPTION
**Pull request by isaacakakpo1**
[TELAPPS-4593 ](https://telnyx.atlassian.net/browse/TELAPPS-4593)
---

Update Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -510,11 +510,86 @@ extension AppDelegate : CXProviderDelegate {
...
 
+### Best Practices when Using PushNotifications with Callkit.
+
+1. When receiving calls from push notifications, it is always required to wait for the connection to the WebSocket before fulfilling the call answer action. This can be achieved by implementing the CXProviderDelegate in the following way (SDK version >=0.1.11):
+```Swift
+func provider(_ provider: CXProvider, perform action: CXStartCallAction) {
+      self.telnyxclient?.answerFromPush(answerAction:action)
+}
+```
+When the `answerFromPush(answerAction:action)` is called. Callkit sets the Call state to `connecting` to alert the user the call is
```
Comment (RJSkorski) [inline README.md:521]: Please correct it:
When the answerFromPush(answerAction: action) is called, Callkit sets the call state to 'connecting' to alert the user that the call is being connected. Once the call is active, the timer starts.

Comment (isaacakakpo1) [reply README.md]: Updated

File: README.md [hunk 2]
```diff
@@ -510,11 +510,86 @@ extension AppDelegate : CXProviderDelegate {
...
+     
+        self.telnyxClient?.endCallFromCallkit(endAction:action)
+     
+}
+```
+   Calling this method solves the race condition, where call is ended before the client connects to the webserver. This way the call is
+   ended on the callee side once a connection is established.
+   
+3. Debugging Push Notification Call Issues : Logs on the receivers end are neccessary to fully debug issues relating to push notifications.
+   However the debugger is not attached when app is completely killed. To solve for this the app can simply be put in background,voip push notifications
```
Comment (RJSkorski) [inline README.md:580]: is that correct from tech perspective? 
Logs on the receiver's end are necessary to thoroughly debug issues related to push notifications. However, the debugger is not attached when the app is completely killed. To address this, the app can simply be placed in the background. VOIP push notifications should then come through, and the debugger should record all logs. You can filter logs using the keyword 'TxClient' to isolate all SDK-related messages

Comment (isaacakakpo1) [reply README.md:580]: Yes this is correct. Every-time we ask for logs. They say it's not possible to get logs on fro the callee. This is a workaround for that

Comment (RJSkorski) [reply README.md:580]: Can you use that text?

**Review by RJSkorski — APPROVED** [review]

**Pull request merged**